js/index.js

```diff
@@ -1,61 +1,97 @@
-let listBooks = [];
-// Local Storage
-const saveToLocalStorage = () => {
-  localStorage.setItem('Library', JSON.stringify(listBooks));
-};
-
-const getDataFromLocalStorage = () => {
-  const data = JSON.parse(localStorage.getItem('Library'));
-  if (data !== null) {
-    listBooks = data;
-  }
-};
-
-function displayBooks() {
-  const section = document.querySelector('#book-list');
-  let books = '';
-  getDataFromLocalStorage();
-
-  listBooks.forEach((book, index) => {
-    books += `<article class="book">
-    <p>${book.title} <br> ${book.author}</p>
-    <button type="button" id="${index}" class="remove-btn" onclick="removeBook(${index})">Remove</button>
-    <hr>
-  </article>`;
-  });
-
-  if (listBooks.length === 0) {
-    books = '<p>Library is empty...</p>';
-  }
-  section.innerHTML = books;
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
+
```

Comment:
- [ ] Please don't disable the linter errors, instead fix them manually. It will help you to write clean code.